draft/oss-support/post.md

```diff
@@ -0,0 +1,82 @@
+# Are We Taking OpenSource For Granted?
+
+Free software. It's not _really_ free. Time, effort, innovation and labor go
+into developing the open source projects that we use and sometimes can't live
+without. A lot of these projects live in the background. Many are known only to
+those that look under the hood. But these code bases are the building blocks
+of the net; the "dependencies" in our apps and OSs (see the EULA of OSX and
+you'll see open source projects listed). Many open source projects find
+funding through sales of support, licenses, and by hosting. But quite a few are
+just put out there and sometimes they find their way into the tool boxes of
+devs and designers, to the point where they almost become ubiquitous.
+
+When you think about it, how many of the open source projects that you use all
+the time are not heavily funded operations? VI, Emacs, heck even tools like
+[sed][link_sed] to niche languages like [Tcl][link_tcl] or [Lua][link_lua].
```

Comment:
> VI, Emacs, heck even tools like sed to niche languages like Tcl or Lua

This is not a complete sentence. Maybe rework this a bit? Could combine it somehow with the previous one.